<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManuallyTaggerWindow</class>
 <widget class="QMainWindow" name="ManuallyTaggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="push_back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="tags_container">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>653</width>
            <height>437</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_3"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScroll">
   <property name="text">
    <string>Scroll</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionScroolBack">
   <property name="text">
    <string>ScroolBack</string>
   </property>
   <property name="shortcut">
    <string>Shift+Space</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>zoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>zoomOut</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>next</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionScrollUp">
   <property name="text">
    <string>scrollUp</string>
   </property>
   <property name="toolTip">
    <string>scroll up</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionScrollDown">
   <property name="text">
    <string>scrollDown</string>
   </property>
   <property name="toolTip">
    <string>scroll down</string>
   </property>
   <property name="shortcut">
    <string>J</string>
   </property>
  </action>
  <action name="actionScrollLeft">
   <property name="text">
    <string>scrollLeft</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionScrollRight">
   <property name="text">
    <string>scrollRight</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
